apps/api/src/domain/models/shared/common-commands/grant-resource-read-access-to-user/resource-read-access-granted-to-user.event.ts

```diff
@@ -12,5 +12,5 @@ export type ResourceReadAccessGrantedToUserPayload = {
 
 @CoscradEvent(`RESOURCE_READ_ACCESS_GRANTED_TO_USER`)
 export class ResourceReadAccessGrantedToUser extends BaseEvent<ResourceReadAccessGrantedToUserPayload> {
-    type = RESOURCE_READ_ACCESS_GRANTED_TO_USER;
+    readonly type = RESOURCE_READ_ACCESS_GRANTED_TO_USER;
```

Comment:
Should `RESOURCE_READ_ACCESS_GRANTED_TO_USER` be a string rather than a constant?

Reply:
Yea, that's a good call. Done.